react-task/src/components/Register.jsx

```diff
@@ -15,28 +15,86 @@ const Register = () => {
     "Uagadou School of Magic ",
   ];
 
+  const [registerData, setRegisterData] = useState({
+    firstName: "",
+    lastName: "",
+    email: "",
+    username: "",
+    dateOfBirth: "",
+    password: "",
+    confirmPassword: "",
+    school: "",
+  });
+
+  const handleDataChange = (newValue) => {
```

Comment:
As per previous comment

```suggestion
  const handleChange = (newValue) => {
```